tests/features/support/step-runners.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file Nightwatch interactions
+ * @see {@link http://nightwatchjs.org/api#expect-api|Nightwatch Expect assertions}
+ */
+
+const { client } = require('nightwatch-api');
+const { europeanaId } = require('./europeana-identifiers.js');
+const { url } = require('../config/nightwatch.conf.js').test_settings.default.globals;
+
+const pages = {
+  'home page': `${url}/`,
+  'search page': `${url}/search`,
+  'record page': `${url}/record${europeanaId()}`,
+  'first page of results': `${url}/search?query=&page=1`
+};
+
+function qaSelector(qas) {
+  return [qas]
+    .flat()
+    .map(qa => qa ? `[data-qa="${qa}"]` : '')
+    .reverse()
+    .join(' ');
+}
+
+function pageUrl(pageName) {
+  return pageName.startsWith('/') ? `${url}${pageName}` : pages[pageName];
+}
+
+module.exports = {
+  openAPage: async function (pageName) {
+    await client.url(pageUrl(pageName));
+  },
+  clickOnTheTarget: async function (qas) {
+    const selector = qaSelector(qas);
+    await client.expect.element(selector).to.be.visible;
+    await client.click(selector);
+  },
+  seeATarget: async function (qas) {
```

Comment:
Are we sure these need to be async? If we want to check for a bunch of elements on the page at the same time, it would be faster to do so synchronously. The risk being that a check happens far too late, but I've never seen them run into the next scenario.

Reply:
Removed from a few of them. All others need it or errors occur.

Total run time is roughly the same as on master. +/- a second depending on the run.